data/src/test/java/org/apache/iceberg/TestSplitScan.java

```diff
@@ -32,59 +35,50 @@
 import org.apache.iceberg.io.FileAppender;
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
 import org.apache.iceberg.types.Types;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-
-@RunWith(Parameterized.class)
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.io.TempDir;
+
+@ExtendWith(ParameterizedTestExtension.class)
 public class TestSplitScan {
   private static final Configuration CONF = new Configuration();
   private static final HadoopTables TABLES = new HadoopTables(CONF);
-
   private static final long SPLIT_SIZE = 16 * 1024 * 1024;
 
   private static final Schema SCHEMA =
       new Schema(
           required(1, "id", Types.IntegerType.get()), required(2, "data", Types.StringType.get()));
 
+  @Parameters(name = "fileFormat = {0}")
```

Comment:
nit: moving stuff around within the file causes unnecessary diffs and makes it more difficult to review. It would be better if this method just stayed at L60+ because all the method needs is just an update to the annotation itself and doesn't require signature changes

Reply:
I moved it back below `expectedRecords`. I hope it's OK.